tensorboard/plugins/histogram/summary_test.py

```diff
@@ -173,7 +173,8 @@ def histogram(self, *args, **kwargs):
 
   def read_single_event_from_eventfile(self):
     event_files = sorted(glob.glob(os.path.join(self.get_temp_dir(), '*')))
-    events = list(tf.compat.v1.train.summary_iterator(event_files[-1]))
+    self.assertEqual(len(event_files), 1)
+    events = list(tf.compat.v1.train.summary_iterator(event_files[0]))
```

Comment:
Do we not want to delete the event file here?

Reply:
Done.  I missed updating this file, thanks for catching this.